src/qt/createwalletdialog.cpp

```diff
@@ -45,7 +46,7 @@ QString CreateWalletDialog::walletName() const
     return ui->wallet_name_line_edit->text();
 }
 
-bool CreateWalletDialog::encrypt() const
+bool CreateWalletDialog::encryptWallet() const
```

Comment:
nit, both previous and current suggest an action. How about `isEncryptChecked` and `isMakeBlankChecked`?

Reply:
Done, updated `disablePrivateKeys` as well for consistency.